packages/nx/src/command-line/nx-commands.ts

```diff
@@ -9,6 +9,7 @@ import { getPackageManagerCommand } from '../utils/package-manager';
 import { writeJsonFile } from '../utils/fileutils';
 import { WatchArguments } from './watch';
 import { runNxSync } from '../utils/child-process';
+import { stripIndents } from '../utils/strip-indents';
```

Comment:
This isn't needed?

Reply:
It's not strictly needed for sure - I currently use it but can just format the message myself if you don't want it imported here